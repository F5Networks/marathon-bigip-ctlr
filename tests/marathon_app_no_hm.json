[
    {
        "acceptedResourceRoles": null,
        "args": [
            "--sse",
            "--marathon",
            "http://10.141.141.10:8080",
            "--partition",
            "mesos",
            "--hostname",
            "10.128.1.145",
            "--username",
            "admin",
            "--password",
            "default"
        ],
        "backoffFactor": 1.15,
        "backoffSeconds": 1,
        "cmd": null,
        "constraints": [],
        "container": {
            "docker": {
                "forcePullImage": false,
                "image": "f5networks/marathon-bigip-ctlr:latest",
                "network": "BRIDGE",
                "parameters": [],
                "privileged": false
            },
            "type": "DOCKER",
            "volumes": []
        },
        "cpus": 0.5,
        "dependencies": [],
        "deployments": [],
        "disk": 0,
        "env": {},
        "executor": "",
        "healthChecks": [],
        "id": "/f5-service-router",
        "instances": 1,
        "labels": {},
        "maxLaunchDelaySeconds": 3600,
        "mem": 128,
        "ports": [
            10000
        ],
        "requirePorts": false,
        "storeUrls": [],
        "tasks": [
            {
                "appId": "/f5-service-router",
                "host": "10.141.141.10",
                "id": "f5-service-router.8f0f064b-1b7c-11e6-b12b-02429f4f0fd6",
                "ports": [
                    31010
                ],
                "slaveId": "da0e67da-31d9-4fbd-8608-ac160ad82f55-S0",
                "stagedAt": "2016-05-16T15:40:47.140Z",
                "startedAt": "2016-05-16T15:40:48.027Z",
                "version": "2016-05-13T18:08:15.204Z"
            }
        ],
        "tasksHealthy": 0,
        "tasksRunning": 1,
        "tasksStaged": 0,
        "tasksUnhealthy": 0,
        "upgradeStrategy": {
            "maximumOverCapacity": 1,
            "minimumHealthCapacity": 1
        },
        "uris": [],
        "user": null,
        "version": "2016-05-13T18:08:15.204Z",
        "versionInfo": {
            "lastConfigChangeAt": "2016-05-13T18:08:15.204Z",
            "lastScalingAt": "2016-05-13T18:08:15.204Z"
        }
    },
    {
        "acceptedResourceRoles": null,
        "args": null,
        "backoffFactor": 1.15,
        "backoffSeconds": 1,
        "cmd": null,
        "constraints": [],
        "container": {
            "docker": {
                "forcePullImage": false,
                "image": "user/node-web-app",
                "network": "BRIDGE",
                "parameters": [],
                "portMappings": [
                    {
                        "containerPort": 8088,
                        "hostPort": 0,
                        "protocol": "tcp",
                        "servicePort": 10001
                    }
                ],
                "privileged": false
            },
            "type": "DOCKER",
            "volumes": []
        },
        "cpus": 0.1,
        "dependencies": [],
        "deployments": [],
        "disk": 0,
        "env": {},
        "executor": "",
        "healthChecks": [
            {
                "gracePeriodSeconds": 5,
                "ignoreHttp1xx": false,
                "intervalSeconds": 20,
                "maxConsecutiveFailures": 3,
                "path": "/",
                "portIndex": 0,
                "protocol": "HTTP",
                "timeoutSeconds": 20
            }
        ],
        "id": "/server-app",
        "instances": 4,
        "labels": {
            "F5_0_BIND_ADDR": "10.128.10.240",
            "F5_0_MODE": "http",
            "F5_0_PORT": "80",
            "F5_PARTITION": "mesos"
        },
        "maxLaunchDelaySeconds": 3600,
        "mem": 16,
        "ports": [
            10001
        ],
        "requirePorts": false,
        "storeUrls": [],
        "tasks": [
            {
                "appId": "/server-app",
                "healthCheckResults": [
                    {
                        "alive": true,
                        "consecutiveFailures": 0,
                        "firstSuccess": "2016-05-18T20:14:20.492Z",
                        "lastFailure": null,
                        "lastSuccess": "2016-05-18T20:21:20.767Z",
                        "taskId": "server-app.0f1279f6-1d35-11e6-b12b-02429f4f0fd6"
                    }
                ],
                "host": "10.141.141.10",
                "id": "server-app.0f1279f6-1d35-11e6-b12b-02429f4f0fd6",
                "ports": [
                    31615
                ],
                "slaveId": "da0e67da-31d9-4fbd-8608-ac160ad82f55-S0",
                "stagedAt": "2016-05-18T20:14:00.472Z",
                "startedAt": "2016-05-18T20:14:01.350Z",
                "version": "2016-05-18T20:14:00.431Z"
            },
            {
                "appId": "/server-app",
                "healthCheckResults": [
                    {
                        "alive": true,
                        "consecutiveFailures": 0,
                        "firstSuccess": "2016-05-18T15:47:56.095Z",
                        "lastFailure": null,
                        "lastSuccess": "2016-05-18T20:21:09.083Z",
                        "taskId": "server-app.d7a3a630-1d0f-11e6-b12b-02429f4f0fd6"
                    }
                ],
                "host": "10.141.141.10",
                "id": "server-app.d7a3a630-1d0f-11e6-b12b-02429f4f0fd6",
                "ports": [
                    31972
                ],
                "slaveId": "da0e67da-31d9-4fbd-8608-ac160ad82f55-S0",
                "stagedAt": "2016-05-18T15:47:36.092Z",
                "startedAt": "2016-05-18T15:47:36.939Z",
                "version": "2016-05-18T15:47:36.046Z"
            },
            {
                "appId": "/server-app",
                "healthCheckResults": [
                    {
                        "alive": true,
                        "consecutiveFailures": 0,
                        "firstSuccess": "2016-05-18T20:14:20.492Z",
                        "lastFailure": null,
                        "lastSuccess": "2016-05-18T20:21:20.767Z",
                        "taskId": "server-app.0f12a107-1d35-11e6-b12b-02429f4f0fd6"
                    }
                ],
                "host": "10.141.141.10",
                "id": "server-app.0f12a107-1d35-11e6-b12b-02429f4f0fd6",
                "ports": [
                    31132
                ],
                "slaveId": "da0e67da-31d9-4fbd-8608-ac160ad82f55-S0",
                "stagedAt": "2016-05-18T20:14:00.473Z",
                "startedAt": "2016-05-18T20:14:01.362Z",
                "version": "2016-05-18T20:14:00.431Z"
            },
            {
                "appId": "/server-app",
                "healthCheckResults": [
                    {
                        "alive": true,
                        "consecutiveFailures": 0,
                        "firstSuccess": "2016-05-18T20:07:15.285Z",
                        "lastFailure": null,
                        "lastSuccess": "2016-05-18T20:21:15.913Z",
                        "taskId": "server-app.11a1f3e2-1d34-11e6-b12b-02429f4f0fd6"
                    }
                ],
                "host": "10.141.141.10",
                "id": "server-app.11a1f3e2-1d34-11e6-b12b-02429f4f0fd6",
                "ports": [
                    31982
                ],
                "slaveId": "da0e67da-31d9-4fbd-8608-ac160ad82f55-S0",
                "stagedAt": "2016-05-18T20:06:55.271Z",
                "startedAt": "2016-05-18T20:06:56.121Z",
                "version": "2016-05-18T20:06:55.241Z"
            }
        ],
        "tasksHealthy": 4,
        "tasksRunning": 4,
        "tasksStaged": 0,
        "tasksUnhealthy": 0,
        "upgradeStrategy": {
            "maximumOverCapacity": 1,
            "minimumHealthCapacity": 1
        },
        "uris": [],
        "user": null,
        "version": "2016-05-18T20:14:00.431Z",
        "versionInfo": {
            "lastConfigChangeAt": "2016-05-18T15:47:36.046Z",
            "lastScalingAt": "2016-05-18T20:14:00.431Z"
        }
    },
    {
        "acceptedResourceRoles": null,
        "args": null,
        "backoffFactor": 1.15,
        "backoffSeconds": 1,
        "cmd": null,
        "constraints": [],
        "container": {
            "docker": {
                "forcePullImage": false,
                "image": "user/node-web-app",
                "network": "BRIDGE",
                "parameters": [],
                "portMappings": [
                    {
                        "containerPort": 8088,
                        "hostPort": 0,
                        "protocol": "tcp",
                        "servicePort": 10002
                    }
                ],
                "privileged": false
            },
            "type": "DOCKER",
            "volumes": []
        },
        "cpus": 0.1,
        "dependencies": [],
        "deployments": [],
        "disk": 0,
        "env": {},
        "executor": "",
        "healthChecks": [],
        "id": "/server-app3",
        "instances": 2,
        "labels": {
            "F5_0_BIND_ADDR": "10.128.10.240",
            "F5_0_MODE": "http",
            "F5_0_PORT": "8089",
            "F5_PARTITION": "mesos"
        },
        "maxLaunchDelaySeconds": 3600,
        "mem": 16,
        "ports": [
            10002
        ],
        "requirePorts": false,
        "storeUrls": [],
        "tasks": [
            {
                "appId": "/server-app3",
                "host": "10.141.141.10",
                "id": "server-app3.18da551f-1d36-11e6-b12b-02429f4f0fd6",
                "ports": [
                    31627
                ],
                "slaveId": "da0e67da-31d9-4fbd-8608-ac160ad82f55-S0",
                "stagedAt": "2016-05-18T20:21:26.378Z",
                "startedAt": "2016-05-18T20:21:27.258Z",
                "version": "2016-05-18T20:21:26.343Z"
            },
            {
                "appId": "/server-app3",
                "host": "10.141.141.10",
                "id": "server-app3.0e713b7e-1d36-11e6-b12b-02429f4f0fd6",
                "ports": [
                    31356
                ],
                "slaveId": "da0e67da-31d9-4fbd-8608-ac160ad82f55-S0",
                "stagedAt": "2016-05-18T20:21:08.913Z",
                "startedAt": "2016-05-18T20:21:09.835Z",
                "version": "2016-05-18T20:21:08.891Z"
            }
        ],
        "tasksHealthy": 0,
        "tasksRunning": 2,
        "tasksStaged": 0,
        "tasksUnhealthy": 0,
        "upgradeStrategy": {
            "maximumOverCapacity": 1,
            "minimumHealthCapacity": 1
        },
        "uris": [],
        "user": null,
        "version": "2016-05-18T20:21:26.343Z",
        "versionInfo": {
            "lastConfigChangeAt": "2016-05-18T20:21:08.891Z",
            "lastScalingAt": "2016-05-18T20:21:26.343Z"
        }
    }
]
