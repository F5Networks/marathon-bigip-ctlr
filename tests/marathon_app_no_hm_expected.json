{"pools": [{"loadBalancingMode": "round-robin", "name": "server-app_80", "members": [{"ratio": 1, "state": "user-up", "rateLimit": 0, "session": "user-enabled", "priorityGroup": 0, "connectionLimit": 0, "address": "10.141.141.10", "port": 31132}, {"ratio": 1, "state": "user-up", "rateLimit": 0, "session": "user-enabled", "priorityGroup": 0, "connectionLimit": 0, "address": "10.141.141.10", "port": 31615}, {"ratio": 1, "state": "user-up", "rateLimit": 0, "session": "user-enabled", "priorityGroup": 0, "connectionLimit": 0, "address": "10.141.141.10", "port": 31972}, {"ratio": 1, "state": "user-up", "rateLimit": 0, "session": "user-enabled", "priorityGroup": 0, "connectionLimit": 0, "address": "10.141.141.10", "port": 31982}], "monitors": ["/mesos/server-app_80_http"]}, {"loadBalancingMode": "round-robin", "name": "server-app3_8089", "members": [{"ratio": 1, "state": "user-up", "rateLimit": 0, "session": "user-enabled", "priorityGroup": 0, "connectionLimit": 0, "address": "10.141.141.10", "port": 31356}, {"ratio": 1, "state": "user-up", "rateLimit": 0, "session": "user-enabled", "priorityGroup": 0, "connectionLimit": 0, "address": "10.141.141.10", "port": 31627}], "monitors": []}], "iapps": [], "l7Policies": [], "virtualServers": [{"profiles": [{"partition": "Common", "name": "http"}, {"partition": "Common", "name": "tcp"}], "destination": "/mesos/10.128.10.240:80", "sourceAddressTranslation": {"type": "automap"}, "name": "server-app_80", "connectionLimit": 0, "ipProtocol": "tcp", "enabled": true, "pool": "/mesos/server-app_80"}, {"profiles": [{"partition": "Common", "name": "http"}, {"partition": "Common", "name": "tcp"}], "destination": "/mesos/10.128.10.240:8089", "sourceAddressTranslation": {"type": "automap"}, "name": "server-app3_8089", "connectionLimit": 0, "ipProtocol": "tcp", "enabled": true, "pool": "/mesos/server-app3_8089"}], "monitors": [{"portIndex": 0, "protocol": "HTTP", "name": "server-app_80_http", "timeoutSeconds": 20, "interval": 20, "intervalSeconds": 20, "ignoreHttp1xx": false, "gracePeriodSeconds": 5, "timeout": 61, "maxConsecutiveFailures": 3, "path": "/", "type": "http"}]}
